From e045af148a580203e47e47276e387fc3958391a5 Mon Sep 17 00:00:00 2001
From: Andrei Borzenkov <arvidjaar@gmail.com>
Date: Sat, 30 Apr 2016 09:15:36 +0300
Subject: [PATCH 015/123] net: reset net->stall in grub_net_seek_real

If we open new connection, we need to reset stall indication, otherwise
nothing will ever be polled (low level code rely on this field being
zero when establishing connection).
---
 grub-core/net/net.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/grub-core/net/net.c b/grub-core/net/net.c
index 942459527..26a1fc3fc 100644
--- a/grub-core/net/net.c
+++ b/grub-core/net/net.c
@@ -1672,6 +1672,7 @@ grub_net_seek_real (struct grub_file *file, grub_off_t offset)
     file->device->net->packs.last = NULL;
     file->device->net->offset = 0;
     file->device->net->eof = 0;
+    file->device->net->stall = 0;
     err = file->device->net->protocol->open (file, file->device->net->name);
     if (err)
       return err;
-- 
2.14.3

